src/Nethermind/Chains/sepolia.json

```diff
@@ -60,7 +60,12 @@
     "eip4895TransitionTimestamp": "0x63FD7D60",
     "eip3855TransitionTimestamp": "0x63FD7D60",
     "eip3651TransitionTimestamp": "0x63FD7D60",
-    "eip3860TransitionTimestamp": "0x63FD7D60"
+    "eip3860TransitionTimestamp": "0x63FD7D60",
```

Comment:
different order of EIP's between holesky and sepolia in Shanghai fork makes it harder to compare